finality-aleph/src/party.rs

```diff
@@ -328,6 +351,7 @@ async fn run_session_as_authority<B, C, BE, SC>(
     // This is a temporary solution -- need to fix this in AlephBFT.
     let _ = exit_aggregator_tx.send(());
     let _ = exit_forwarder_tx.send(());
+    let _ = exit_refresher_tx.send(());
```

Comment:
Can we replace all these `let _ = *tx.send(());` with `if let Err(e) = *tx.send(()) {debug!(target: "afa", "*tx closed before terminating it manually: {}", e)}`? The current solution silences warnings which probably shouldn't be silenced?

Reply:
Sure, I followed the old pattern instead of taking the opportunity to fix it.

Reply:
fixed.